tests/run_scapval.py

```diff
@@ -30,26 +37,39 @@ def parse_args():
     return parser.parse_args()
 
 
+def ds_is_k8s_related(result_path):
+    for prodname in K8S_PRODUCTS:
+        ds_basename_stem = "ssg-{prodname}-ds".format(prodname=prodname)
+        if ds_basename_stem in result_path:
+            return True
+    return False
+
+
+def format_requirement_feedback(req_id, message):
```

Comment:
I think it should be named print_requirement_feedback as its main goal is to print something, it doesn't return a formatted string.